config/FindNDI.cmake

```diff
@@ -76,7 +76,7 @@ endif()
 
 find_path( NDI_INCLUDE_DIR
   NAMES ndi.h
-  HINTS ${NDI_ROOT_DIR}/include/ndi
+  HINTS ${NDI_ROOT_DIR}/include/ndi ${NDI_ROOT_DIR}/include
```

Comment:
@KineticTheory I needed this change to get my local installation path (e.g. `/my/path/to/NDI/install`) to work -- I don't think this would break anything anywhere else? But I wanted to check with you

Reply:
@KineticTheory @brryan I ran into this too - in the previous (2.1.4) NDI release, it looks like the include files were located at both `${NDI_ROOT_DIR}/include/ndi` AND` ${NDI_ROOT_DIR}/include`... however, the additional nested `/ndi/` directory doesn't seem to exist in 2.2.0alpha. - It might be worth simply removing the `/include/ndi` version to see if the `/include/` path works for 2.1.4 on all platforms!

Reply:
There is considerable inconsistency concerning both the include path and the library path.  Before Tommy took over, the NDI team was trying to adopt gnu coding standards that required the extra `ndi` in the include path and the lib directory was renamed `lib64` for 64-bit platforms.  Tommy has since reverted to the earlier installation layout.  I guess we just need to be flexible when looking for NDI.

Reply:
Ahh, okay! Thanks, @KineticTheory, that's good to know!

Reply:
OK sounds good. I'll leave both these HINT paths in and hopefully everything is fine.